irohad/consensus/yac/impl/yac_crypto_provider_impl.hpp

```diff
@@ -19,13 +19,15 @@
 #define IROHA_YAC_CRYPTO_PROVIDER_IMPL_HPP
 
 #include "consensus/yac/yac_crypto_provider.hpp"
+#include "cryptography/keypair.hpp"
```

Comment:
Forward decl seems enough here

Reply:
check the rest as well plz

Reply:
no, there is member initialization

Reply:
Ehm, may you point that place out?

Reply:
https://github.com/hyperledger/iroha/pull/1185/files/7749b45e4bae3ef8e494d9f0949842acc0f7b00f#diff-bfaf89f725240ddf0dbe056db3ef5183R41

Reply:
*Instantination, not initialization

Reply:
Seems like a bug, ok then